src/shell_install/nushell.rs

```diff
@@ -14,15 +14,10 @@
 
 use {
     super::{util, ShellInstall},
-    crate::{
-        conf,
-        errors::*,
-    },
+    crate::{conf, errors::*},
     directories::BaseDirs,
     std::path::PathBuf,
-    termimad::{
-        mad_print_inline,
-    },
+    termimad::mad_print_inline,
```

Comment:
@Canop I had autoformat with cargo activated and I changed this lines by accident, this has nothing to do with this pr, shall I revert it or leave it as it is?

Reply:
It's Ok, don't worry for such a small change, don't lose time reverting.

Reply:
BTW I'd like to use cargo fmt but things are slow to stabilize...
```
Warning: can't set `imports_layout = Vertical`, unstable features are only available in nightly channel.
Warning: can't set `version = Two`, unstable features are only available in nightly channel.
```